Procfile.dev

```diff
@@ -0,0 +1,4 @@
+web: rails s
+hot: sh -c 'cd client && npm start'
+client: sh -c 'rm app/assets/javascripts/generated/* || true && cd client && npm run build:dev:client'
```

Comment:
Note, we have to rebuild the client webpack file with `wepback -w` when files change. See definition [here](https://github.com/shakacode/react_on_rails-generator-results/blob/redux-1_1_1/client/package.json#L31):

```javascript
"build:dev:client": "webpack -w --config webpack.client.rails.config.js",
```